inc/classes/class-imagify-views.php

```diff
@@ -222,7 +222,17 @@ public function add_network_menus() {
 	 * @return array
 	 */
 	public function plugin_action_links( $actions ) {
```

Comment:
IMO, this is a good opportunity to move code to a new architecture more in line with our standards.

We could create a new class in `classes/Admin/`, `Admin.php`, implementing the Subscriber interface, and having `User` as a dependency instead of creating the instance in the method itself.

We can also add a service provider to instantiate it.

That will make things easier to test, and start a foundation for the future of the code related to the admin part